src/object/obj_rpc.h

```diff
@@ -398,7 +398,7 @@ CRT_RPC_DECLARE(obj_ec_agg, DAOS_ISEQ_OBJ_EC_AGG, DAOS_OSEQ_OBJ_EC_AGG)
 	((daos_key_t)		(er_dkey)		CRT_VAR)	\
 	((daos_iod_t)		(er_iod)		CRT_VAR)	\
 	((struct dcs_iod_csums)	(er_iod_csums)		CRT_ARRAY)	\
-	((uint64_t)		(er_epoch)		CRT_VAR)	\
+	((daos_epoch_range_t)	(er_epoch_range)	CRT_VAR)	\
```

Comment:
bump the DAOS_OBJ_VERSION?

Reply:
as it is a pure server-side RPC, so it does not affect RPC inter-operability, do we still need to bump the version?

Reply:
ah, hmm we probably will support rolling upgrade, which might need incompatible support between servers, but this won't happen in 2.4. I think

Reply:
So the effect here is rolling upgrade is not supported in 2.2?

Reply:
> So the effect here is rolling upgrade is not supported in 2.2?

suppose the rolling upgrade is upgrade partial server node, that is not plan to support even for 2.4 I think.